### PR TITLE
fix: duration type

### PR DESCRIPTION
**Pull request by chlumper**
* change to numeric schema type for time intervals (instead of duration)
(MerkleTreeTimeout, MinInterval -> int; SubmissionInterval -> double)
* improve doc (duration units, sampel values)
* remove unused broker type at proof sink

### CODE REVIEW

File: DTDL/base_sink.json [hunk 1]
```diff
@@ -10,15 +10,15 @@
...
+      "description": "The maximum number of values to send as batch in one message to the sink (e.g. 20 (ProofSink), 1024 (ValueSink))."
     },
     {
       "@type": "Property",
       "name": "FlushInterval",
       "schema": "integer",
       "writable": true,
       "displayName": "Flush interval",
-      "description": "The interval in which the values should be flushed from the buffer and send to the API. Regardless if the MaxBatchSize was reached."
+      "description": "The interval in which the values should be flushed from the buffer and send to the API. Regardless if the MaxBatchSize was reached. e.g. 1s"
```
Comment (mayrmax) [inline DTDL/base_sink.json:21]: add hint that value is in seconds and change example to e.g. 1

File: DTDL/base_stream.json [hunk 2]
```diff
@@ -41,27 +41,26 @@
...
       "@type": "Property",
       "name": "MerkleTreeTimeout",
-      "schema": "duration",
+      "schema": "integer",
       "writable": true,
-      "displayName": "Merkle Tree Timeout",
-      "description": "Timeout for a single merkle-tree for proofs in ms.",
-      "comment": "If a stream doesn't provides enought values to build an entire merkle tree with the given depth in the given period it will be splitted into smaller merkle-trees."
+      "displayName": "Max Merkle Tree Age",
+      "description": "Maximum merkle tree age in seconds (e.g. 600s).",
```
Comment (mayrmax) [inline DTDL/base_stream.json:54]: e.g. 600

File: DTDL/base_stream.json [hunk 3]
```diff
@@ -41,27 +41,26 @@
...
     {
       "@type": "Property",
       "name": "SubmissionInterval",
-      "schema": "duration",
+      "schema": "double",
       "writable": true,
       "displayName": "Submission Interval",
-      "description": "Timeout for a single merkle-tree for proofs.",
-      "comment": "If a stream doesn't provides enought values to build an entire merkle tree with the given depth in the given period it will be splitted into smaller merkle-trees."
+      "description": "The estimated data submission interval/frequency of the stream in Hz (e.g. 1.2Hz)."
```
Comment (mayrmax) [inline DTDL/base_stream.json:63]: e.g. 1.2

File: DTDL/data_api_value_sink.json [hunk 4]
```diff
@@ -51,23 +51,23 @@
...
+            "description": "Number of maximum retries for calls at Data-API (e.g. 1024)."
         },
         {
             "@type": "Property",
             "name": "RetryInterval",
             "schema": "integer",
             "writable": true,
             "displayName": "Retry interval",
-            "description": "Delay interval for the first retry of call to Data-API on error"
+            "description": "Delay interval (in seconds) for the first retry of call to Data-API on error (e.g. 1s)."
```
Comment (mayrmax) [inline DTDL/data_api_value_sink.json:62]: e.g. 1

File: DTDL/data_api_value_sink.json [hunk 5]
```diff
@@ -51,23 +51,23 @@
...
         },
         {
             "@type": "Property",
             "name": "MaxRetryInterval",
             "schema": "integer",
             "writable": true,
             "displayName": "Max retry interval",
-            "description": "Maximum interval for delays of calls at Data-API on error"
+            "description": "Maximum interval (in seconds) for delays of calls at Data-API on error (e.g. 30s)"
```
Comment (mayrmax) [inline DTDL/data_api_value_sink.json:70]: same

File: DTDL/opcua_source.json [hunk 6]
```diff
@@ -3,23 +3,23 @@
...
     },
     {
       "@type": "Property",
       "name": "PublishingInterval",
       "schema": "integer",
       "writable": true,
-      "displayName": "OPCUA publishing interval",
-      "description": "The opcua publishing interval"
+      "displayName": "OPC UA publishing interval",
+      "description": "The OPC UA publishing interval in milliseconds (e.g. 100ms)."
```
Comment (mayrmax) [inline DTDL/opcua_source.json:22]:
```suggestion
      "description": "The OPC UA publishing interval in milliseconds (e.g. 100)."
```

File: DTDL/proof_sink.json [hunk 7]
```diff
@@ -10,32 +10,32 @@
             "name": "BaseUrl",
             "schema": "string",
             "writable": true,
-            "displayName": "API Url",
-            "description": ""
+            "displayName": "Data-API Url",
```
Comment (mayrmax) [inline DTDL/proof_sink.json:13]:
```suggestion
            "displayName": "Trust-API Url",
```

File: DTDL/proof_sink.json [hunk 8]
```diff
@@ -10,32 +10,32 @@
...
+            "description": "URL of the Data-API (e.g. https://data-api.your-node-name.dataspace-node.com)"
```
Comment (mayrmax) [inline DTDL/proof_sink.json:14]:
```suggestion
            "description": "URL of the Trust-API (e.g. https://trust-api.your-node-name.dataspace-node.com)"
```

File: DTDL/proof_sink.json [hunk 9]
```diff
@@ -10,32 +10,32 @@
...
+            "description": "OAuth scope (e.g. \"profile email data-api node-id\")"
         },
         {
             "@type": "Property",
             "name": "ClientId",
             "schema": "string",
             "writable": true,
             "displayName": "Client id for authentication",
-            "description": ""
+            "description": "Client Id used for authenticaiton (e.g. \"data-api\")"
```
Comment (mayrmax) [inline DTDL/proof_sink.json:38]:
```suggestion
            "description": "Client Id used for authenticaiton (e.g. \"trust-api\")"
```

File: DTDL/proof_sink.json [hunk 10]
```diff
@@ -45,44 +45,21 @@
...
+            "description": "Number of maximum retries for calls at Trust-API (e.g. 1024)."
         },
         {
             "@type": "Property",
             "name": "MaxRetryInterval",
             "schema": "integer",
             "writable": true,
             "displayName": "Max retry interval",
-            "description": "Maximum interval for delays of calls at Trust-API on error"
+            "description": "Maximum interval (in seconds) for delays of calls at Trust-API on error (e.g. 30s)"
```
Comment (mayrmax) [inline DTDL/proof_sink.json:62]:
```suggestion
            "description": "Maximum interval (in seconds) for delays of calls at Trust-API on error (e.g. 30)"
```

File: DTDL/rabbitmq_source.json [hunk 11]
```diff
@@ -11,23 +11,23 @@
...
+            "description": "The connection string for the RabbitMQ value source (e.g. \"amqp://user:pass@host:10000/vhost\")."
         },
         {
             "@type": "Property",
             "name": "MaxBatchSize",
             "schema": "integer",
             "writable": false,
             "displayName": "Max batch size",
-            "description": "The maximum number of values to send as batch in one message to the EdgeAgent."
+            "description": "The maximum number of values to send as batch in one message to the EdgeAgent. e.g. 1024"
```
Comment (mayrmax) [inline DTDL/rabbitmq_source.json:22]:
```suggestion
            "description": "The maximum number of values to send as batch in one message to the EdgeAgent. (e.g. 1024)"
```

File: DTDL/rabbitmq_source.json [hunk 12]
```diff
@@ -11,23 +11,23 @@
...
         },
         {
             "@type": "Property",
             "name": "FlushInterval",
             "schema": "integer",
             "writable": false,
             "displayName": "Flush interval",
-            "description": "The interval in which the values should be flushed from the buffer and send to the Data-API. Regardless if the MaxBatchSize was reached."
+            "description": "The interval (in seconds) in which the values should be flushed from the buffer and send to the Data-API. Regardless if the MaxBatchSize was reached. e.g. 1s"
```
Comment (mayrmax) [inline DTDL/rabbitmq_source.json:30]:
```suggestion
            "description": "The interval (in seconds) in which the values should be flushed from the buffer and send to the Data-API. Regardless if the MaxBatchSize was reached. (e.g. 1)"
```

File: DTDL/replay_value_source.json [hunk 13]
```diff
@@ -20,15 +20,15 @@
...
+            "description": "The name of the CSV file to use (e.g. \"Data/demo_data.csv\")."
         },
         {
             "@type": "Property",
             "name": "Frequency",
             "schema": "double",
             "writable": true,
             "displayName": "Frequency",
-            "description": "The interval to use for replaying values line by line in ms."
+            "description": "The interval (in milliseconds) to use for replaying values line by line in (e.g. 100ms)"
```
Comment (mayrmax) [inline DTDL/replay_value_source.json:31]:
```suggestion
            "description": "The interval (in milliseconds) to use for replaying values line by line in (e.g. 100)"
```

File: DTDL/simulated_value_source.json [hunk 14]
```diff
@@ -11,15 +11,15 @@
...
+            "description": "The maximum number of values to send as batch in one message to the EdgeAgent (e.g. 1024)"
         },
         {
             "@type": "Property",
             "name": "FlushInterval",
             "schema": "integer",
             "writable": true,
             "displayName": "Flush interval",
-            "description": "The interval in which the values should be flushed from the buffer and send to the Data-API. Regardless if the MaxBatchSize was reached."
+            "description": "The interval (in seconds) in which the values should be flushed from the buffer and send to the Data-API. Regardless if the MaxBatchSize was reached. e.g. 1s"
```
Comment (mayrmax) [inline DTDL/simulated_value_source.json:22]:
```suggestion
            "description": "The interval (in seconds) in which the values should be flushed from the buffer and send to the Data-API. Regardless if the MaxBatchSize was reached. (e.g. 1)"
```

File: DTDL/simulated_value_stream.json [hunk 15]
```diff
@@ -36,26 +36,26 @@
...
+            "description": "Maximum for generated values (e.g. 3.25 for double, 1204 for int or 512 for binary)."
         },
         {
             "@type": "Property",
             "name": "Frequency",
             "schema": "double",
             "writable": true,
             "displayName": "Frequency",
-            "description": "Data Frequency in Hz (1/s) "
+            "description": "Data Frequency in Hz (1/s) (e.g 1.0Hz)"
```
Comment (mayrmax) [inline DTDL/simulated_value_stream.json:58]:
```suggestion
            "description": "Data Frequency in Hz (1/s) (e.g 1.0)"
```

**Review by mayrmax — APPROVED** [review]

**Comment by simonpfeifhofer** [comment]
@chlumper great work.

**Review by simonpfeifhofer — APPROVED** [review]

**Pull request merged**